### PR TITLE
Pull request for Branch Desktop version

### PR DESCRIPTION
**Pull request by EzekielUtshudi**
Add comments 
fix stylint errors

### CODE REVIEW

**Review by Hope1226 — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>,

We appreciate your effort in doing your best in this project!
However, there are some issues that you still need to work on to go to the next project. Do not worry we will get you there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Required Changes ♻️
### Project requirements
- [ ] Please kindly make sure your `CSS` is properly linked to the `index.html`. Currently, your page is not reading the `style guides` from `style.css`. It is so because your path is broken, refer to the code below from your `index.html` file:

https://github.com/EzekielUtshudi/Porfolio-Project/blob/f04d56e36957fb83375e69597c9cc773405bbdb7/index.html#L8

CSS file is located in `Css` folder not `css`. Please kindly take care of it :pray:

- [ ] Please kindly make sure you know how to work with `Media query`. I can see that you are using several of them in your `CSS`. Please note that for this project, you need only `mob-version` (which you have already built) and `desktop version` meaning that you need to have only one `breakpoint`.
- Please kindly remove all the following `breakpoints`.

https://github.com/EzekielUtshudi/Porfolio-Project/blob/f04d56e36957fb83375e69597c9cc773405bbdb7/Css/style.css#L431

https://github.com/EzekielUtshudi/Porfolio-Project/blob/f04d56e36957fb83375e69597c9cc773405bbdb7/Css/style.css#L468

https://github.com/EzekielUtshudi/Porfolio-Project/blob/f04d56e36957fb83375e69597c9cc773405bbdb7/Css/style.css#L484

https://github.com/EzekielUtshudi/Porfolio-Project/blob/f04d56e36957fb83375e69597c9cc773405bbdb7/Css/style.css#L484

- You need to understand when to use `min-width` and `max-width`.

_hints: I am going to put it in a simple way when you are building a website using the `mobile-first approach` you use `min-width` with the `necessary breakpoint` when it is `desktop first approach` you use `max-width`. Since you have built the mobile version first, it means you are using a `mobile-first approach`, so you need to stick to `min-width`. As requested by the project your `breakpoint` for switching to the `desktop version` should be in a range of   `992px - 1100px` sample code:_

```CSS
@media (min-width: 1000px) {
……adjust your style here….
}
```

More on that [here](https://www.w3schools.com/css/css_rwd_mediaqueries.asp)

- [ ] Please kindly make sure your design is compliant with the template chosen from [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)
- Headline section:

| The design you implemented  | Original design in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4) |
| ------------- | ------------- |
| Screenshot from 2022-02-17 16-14-30 | Screenshot from 2022-02-17 16-14-14 |

_Actions required:_
- Set up respective background `image`
- Set up respective `icons` for your list of social networks, correct the path if it is broken.

https://github.com/EzekielUtshudi/Porfolio-Project/blob/f04d56e36957fb83375e69597c9cc773405bbdb7/index.html#L42-L88

- Works and About me sections:

| The design you implemented  | Original design in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4) |
| ------------- | ------------- |
| Screenshot from 2022-02-17 16-13-34 | Screenshot from 2022-02-17 16-13-56 |

_Actions required:_
- Please kindly make sure `works` `about` sections are  horizontally centred
https://github.com/EzekielUtshudi/Porfolio-Project/blob/f04d56e36957fb83375e69597c9cc773405bbdb7/index.html#L94-L95

https://github.com/EzekielUtshudi/Porfolio-Project/blob/f04d56e36957fb83375e69597c9cc773405bbdb7/index.html#L243-L245

- contacts section:

| The design you implemented  | Original design in [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4) |
| ------------- | ------------- |
| Screenshot from 2022-02-17 16-15-06 | Screenshot from 2022-02-17 16-39-58 |

_Actions required:_
- Set up respective background `image`
- Set the `layout` just the way it is shown in the template  

-[ ]Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
n/a
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Css/style.css [hunk 1]
```diff
@@ -366,153 +460,164 @@ img {
...
+  .works-item {
+    width: 70%;
+    margin: 0 auto;
   }
+}
 
-  input:focus-visible,
-  textarea:focus-visible {
-    outline-color: #27c080;
+@media (min-width: 768px) {
```
Comment (ReshyResh) [inline Css/style.css:506]: - [ ] As the previous reviewer pointed out, you have added too many breakpoints which weren't required by the project. It is not a bad feature, but the project guidelines explicitly require the desktop breakpoint to be in the range of `992px` - `1100px`, please adjust your desktop breakpoint to have a correct value.

**Review by ReshyResh — CHANGES_REQUESTED** [review]
# Hi @EzekielUtshudi , 

### You've done a great work on making the changes requested by the previous reviewer, and the project looks awesome 👏 
#### You've done well implementing some of the requested changes, but there are still some which aren't addressed yet, mostly regarding the media queries.

 ## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Css/style.css (hunk 1 above)
Comment (EzekielUtshudi) [reply Css/style.css:506]: image

**Review by ReshyResh — APPROVED** [review]
# Hi @EzekielUtshudi , 

### Good job addressing the changes requested previously!
#### Your project is complete!  There is nothing else to say other than... it's time to merge it!
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**